src/main/daml/ContingentClaims/Core/Util/Recursion.daml

```diff
@@ -19,11 +19,11 @@ module ContingentClaims.Core.Util.Recursion (
   , synthesize
 ) where
 
-import Prelude hiding (sequence, mapA, sum)
-import DA.Traversable (Traversable, sequence, mapA)
+import Prelude hiding (mapA, sequence, sum)
```

Comment:
shouldn't import of `Prelude` go at the end?

Reply:
absolutely